### ISSUE TITLE
a11y - default text block minimum contrast and focus - cms ui

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#5203]
**Describe the bug**
The empty fields have a 'Type text...' placeholder, but the text contrast is very low, and the field lacks borders/focus, making it less noticeable. In general, light gray text on white background should be avoided.

**To Reproduce**

1. Go to https://demo.plone.org/edit
2. enter in edit mode by clicking on the edit button (top-left)
3. press "enter"

**Expected behavior**
- The placeholder text color must have an acceptable minimum contrast.
- The input field created by the text block needs to have a visible focus when focused.

**Screenshots**
<img width="1340" alt="Schermata 2023-09-20 alle 15 49 41" src="https://github.com/plone/volto/assets/60133113/408dca96-a826-457a-b3c6-2abcde51b6db">

**Comment by YashGupt29** [comment]
okay so may i work on this issue

**Comment by stevepiercy** [comment]
@YashGupt29 please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html).

**Comment by Maverick073** [comment]
hey I see this issue is still open and I guess no one's is working , so shall I take and work upon this @Wagner3UB ?

**Comment by stevepiercy** [comment]
@Maverick073 please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).
